pkg/openapi/openapi_generated.go

```diff
@@ -25960,7 +25960,7 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 						},
 						"enableTLSAuth": {
 							SchemaProps: spec.SchemaProps{
-								Description: "EnableTLSAuth indicats client and peer TLS auth should be enforced",
+								Description: "EnableTLSAuth indicators client and peer TLS auth should be enforced",
```

Comment:
Probably `indicates`

Reply:
I'd fixed in the source but needed to regenerate the openapi